vispy/plot/plotwidget.py

```diff
@@ -222,7 +222,7 @@ def image(self, data, cmap='cubehelix', clim='auto', fg_color=None):
 
     def mesh(self, vertices=None, faces=None, vertex_colors=None,
              face_colors=None, color=(0.5, 0.5, 1.), fname=None,
-             meshdata=None):
+             meshdata=None, shading=None):
```

Comment:
The problem with the 'smooth' default is that it prevents face_colors from working. Sure, people can explicitly set smooth=None to enable face_colors, but this might be confusing to some people if they try to modify the face_colors and see no effect.
However, if people think that's ok, It's not a big deal to me.
Let me know, and I'll modify the code accordingly.